collection/collection/build.gradle

```diff
@@ -17,16 +17,19 @@
 
 
 import androidx.build.AndroidXComposePlugin
+import androidx.build.JetbrainsAndroidXPlugin
 import androidx.build.LibraryType
 
 plugins {
     id("AndroidXPlugin")
 
     //  TODO move all functionality to Compose-independent plugin (`collection` shouldn't depend on Compose concepts)
     id("AndroidXComposePlugin")
```

Comment:
Do we still need `AndroidXComposePlugin` here?

Reply:
Yes. We still need it until we implement androidx-like (build) API for targets declaration and configuration. It would be a separate step as we dicussed earlier.